wagtailio/static/sass/components/_code-snippet.scss

```diff
@@ -0,0 +1,82 @@
+.code-snippet {
+    $root: &;
+    background-color: $color--indigo;
+    color: $color--white;
+    display: flex;
+    align-items: center;
+    padding: 15px 20px;
+    border-radius: 6px;
+    position: relative;
+    grid-column: 2 / span 2;
+    margin-bottom: 15px;
+    font-size: 0.9375rem;
```

Comment:
Maybe one to add to the tech debt, we could refactor these into vars so we can better control/know what's actively used on the site?

Reply:
Yeah definitely. I did it by eye for mobile as we have no design.